iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/load/LoadTsFileManager.java

```diff
@@ -435,6 +437,27 @@ private void loadAll(boolean isGeneratedByPipe, ProgressIndex progressIndex)
                           dataRegion.getDataRegionId(),
                           Tag.TYPE.toString(),
                           Metric.LOAD_POINT_COUNT.toString());
+                  // Because we cannot accurately judge who is the leader here,
+                  // we directly divide the writePointCount by the replicationNum to ensure the
+                  // correctness of
+                  // this metric, which will be accurate in most cases
```

Comment:
```suggestion
                  // Because we cannot accurately judge who is the leader here,
                  // we directly divide the writePointCount by the replicationNum to ensure the
                  // correctness of this metric, which will be accurate in most cases           
```